java/Yaroslav-Dzhur/src/com/zoolatech/lecture1/tasks/_4/conditionalOperator.java

```diff
@@ -0,0 +1,23 @@
+package com.zoolatech.lecture1.tasks._4;
+
+/*
+Write a program that accepts a day of the week as an integer number (1 - Monday, 2 - Tuesday, …, 7 - Sunday)
+and output “Need to go to work…” if the day is a weekday or “Sleeping…” if it’s a weekend.
+The conditional operator version.
+*/
+
+import java.util.Scanner;
+
+public class conditionalOperator {
```

Comment:
The class name should be defined using the upper camel case: ConditionalOperator. Same for other similar classes